.github/DISCUSSION_TEMPLATE/ideas.yml

```diff
@@ -0,0 +1,10 @@
+body:
+- type: markdown
+  attributes:
+    value: |
+      > Note: For ideas not related to the CLI or API, please go to the [Vercel Community](https://github.com/orgs/vercel/discussions/categories/ideas)
+- type: textarea
+  attributes:
+    label: Idea
+  validations:
+    required: true
```

Comment:
I see you only added 3 of the 6 types as templates.

Should we disable all of these types so there is only one button?

Or should we add all 6 types as templates?

<img width="1245" alt="image" src="https://user-images.githubusercontent.com/229881/224828230-8ad3082d-8daa-434c-b77c-442f82578d34.png">

Reply:
I picked the ones that were most commonly used, but I can add it for them all if that seems more appropriate

Reply:
Done 😄